docs/src/main/sphinx/object-storage/file-system-azure.md

```diff
@@ -1,11 +1,12 @@
 # Azure Storage file system support
 
 Trino includes a native implementation to access [Azure
-Storage](https://learn.microsoft.com/en-us/azure/storage/) with a catalog using
-the Delta Lake, Hive, Hudi, or Iceberg connectors.
+Storage](https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction)
+with a catalog using the Delta Lake, Hive, Hudi, or Iceberg connectors.
 
 Enable the native implementation with `fs.native-azure.enabled=true` in your
-catalog properties file.
+catalog properties file. Additionally, the storage account must be ADLS GEN2 and
```

Comment:
thats weird wording .. "must be" .. if we only support ADLS Gen 2 then we should just add that to the first paragraph .. and maybe as part of that we should actually link to https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-overview#about-azure-data-lake-storage-gen2

maybe confirm with @dain (implemented that support from what I recall) .. or @electrum  (file system lead)